query/query.ts

```diff
@@ -160,11 +161,22 @@ export class QueryArrayResult<T extends Array<unknown> = Array<unknown>>
   }
 }
 
-export class QueryObjectResult<
-  T = Record<string, unknown>,
-> extends QueryResult {
+export class QueryObjectResult< T = Record<string, unknown>,> extends QueryResult {
   public rows: T[] = [];
 
+  private snakeToCamelCase = (input: string) =>
```

Comment:
This breaks for `trailing_underscores_`.

Reply:
how can I reproduce this breaking? lint doesn't complain on query.ts using following as config file: https://deno.land/x/deno@v1.14.2/cli/schemas/config-file.v1.json
Thanks